apps/dashboard/src/utils/auth.ts

```diff
@@ -6,5 +8,9 @@ declare namespace Clerk {
 }
 
 export async function getToken(): Promise<string> {
+  if (MODE === 'test') {
+    return localStorage.getItem('nv_auth_token') ?? '';
+  }
```

Comment:
For tests, we store the API session-generated JWT token in the local storage because we don't use the one that is provided by the Clerk as the token signature won't match on the API, and we will not be able to authorize the requests. 

In short the `@novu/testing` library needs some refactoring which is out of scope and might take a lot of time.